puphpet/config.yaml.template

```diff
@@ -208,13 +264,20 @@ python:
 mysql:
     install: '1'
     override_options: {  }
-    root_password: '123'
+    root_password: password
```

Comment:
Cool. I'm done now. Thanks for your patience through all this. I appreciate your willingness to edit this PR base on my inane comments.